shapely/geometry/geo.py

```diff
@@ -90,8 +113,30 @@ def shape(context):
 
 
 def asShape(context):
-    """Adapts the context to a geometry interface. The coordinates remain
-    stored in the context.
+    """
+    Adapts the context to a geometry interface. The coordinates remain
+    stored in the context, and changes to them will be reflected in the
+    returned geometry object.
+
+    Parameters
+    ----------
+    context :
+        a GeoJSON-like dict, which provides a "type" member describing the type
+        of the geometry and "coordinates" member providing a list of coordinates,
+        or an object which implements __geo_interface__.
+
+    Example
+    -------
+    Create a Point from GeoJSON, and then create a copy using __geo_interface__.
```

Comment:
Before this gets merged I need to alter this description - I copied it from the example for `shape` and then proceeded to not update it.